dbhydra/src/abstract_table.py

```diff
@@ -237,11 +239,16 @@ def __init__(self, db1, name, columns = None, types: Optional[list[str]] = None,
     # Temporary disabled, please make sure this is implemented where needed, don't introduce breaking changes please
     # @abc.abstractmethod
     @classmethod
-    def init_from_column_type_dict(cls, db1, name, column_type_dict, id_column_name="id"):
+    def init_from_column_type_dict(cls, db1, name, column_type_dict, id_column_name="id", use_csv=False):
```

Comment:
could we avoid this?

Reply:
I would like the signature of the function to be the same for all dialects

Reply:
Didn't find a quick way to do that, tried also to have an overloaded method for XlsxTable but that did not solve the problem